### PR TITLE
Laboratory work #8, Smirnova Anastasia - 20FPL1

### PR DESCRIPTION
**Pull request by Smirnova23**

### CODE REVIEW

File: lab_7_llm/main.py [hunk 1]
```diff
@@ -192,4 +192,4 @@ def run(self) -> dict | None:
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py:192]: Checkout the files from 7 lab to the master

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+datasets==2.16.1
+pandas==2.1.4
+torch==2.1.2
+torchinfo==1.8.0
+transformers==4.38.2
```
Comment (SalnikovIgor) [inline requirements.txt]: New line

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Fix comments and tests

File: requirements.txt (hunk 2 above)
Comment (Smirnova23) [reply requirements.txt]: fixed comments and tests

**Pull request closed without merging**